distro.json

```diff
@@ -243,5 +271,31 @@
         }
       }
     }
+  },
+  {
+    "name": "openapi_generator",
+    "platforms": {
+      "all": {
+        "universal": {
+          "url": "http://nxxm.indigenious.io/distro/v.0.0.15/openapi_generator/openapi-generator.zip",
```

Comment:
Which version of openapi-generator is that ? could you encode it in the name ?

Reply:
If I store the scripts on GitHub I won't be able to have the version number in the zip name. 
It's up to you which one you prefer to store under GitHub or have a zip with the version number.

Reply:
We are not speaking of the same package, I meant only the script on GH and the openapi-generator tool on our server.

Reply:
oki, It works I do this right after the ftxui